spec/index.bs

```diff
@@ -1911,33 +1911,37 @@ interface GPU {
         chooses according to the provided options.
 
         <div algorithm=GPU.requestAdapter>
-            **Called on:** {{GPU}} |this|.
+            <div data-timeline=content>
+                **Called on:** {{GPU}} |this|.
 
-            **Arguments:**
+                **Arguments:**
 
-            <pre class=argumentdef for="GPU/requestAdapter(options)">
-                |options|: Criteria used to select the adapter.
-            </pre>
+                <pre class=argumentdef for="GPU/requestAdapter(options)">
+                    |options|: Criteria used to select the adapter.
+                </pre>
 
-            **Returns:** {{Promise}}&lt;{{GPUAdapter}}?&gt;
+                **Returns:** {{Promise}}&lt;{{GPUAdapter}}?&gt;
 
-            1. Let |promise| be [=a new promise=].
-            1. Issue the following steps on the [=Device timeline=] of |this|:
+                [=Content timeline=] steps:
 
-                <div class=device-timeline>
-                    1. If the user agent chooses to return an adapter, it should:
-                        1. Create a [=valid=] [=adapter=] |adapter|, chosen according to
-                            the rules in [[#adapter-selection]] and the criteria in |options|.
+                1. Let |promise| be [=a new promise=].
+                1. Issue the |initialization steps| on the [=Device timeline=] of |this|.
+                1. Return |promise|.
+            </div>
+            <div data-timeline=device>
+                [=Device timeline=] |initialization steps|:
 
-                        1. If |adapter| meets the criteria of a [=fallback adapter=] set
-                            |adapter|.{{adapter/[[fallback]]}} to `true`.
+                1. If the user agent chooses to return an adapter, it should:
+                    1. Create a [=valid=] [=adapter=] |adapter|, chosen according to
+                        the rules in [[#adapter-selection]] and the criteria in |options|.
 
-                        1. [=Resolve=] |promise| with a new {{GPUAdapter}} encapsulating |adapter|.
+                    1. If |adapter| meets the criteria of a [=fallback adapter=] set
+                        |adapter|.{{adapter/[[fallback]]}} to `true`.
 
-                    1. Otherwise, |promise| [=resolves=] with `null`.
-                </div>
-            1. Return |promise|.
+                    1. [=Resolve=] |promise| with a new {{GPUAdapter}} encapsulating |adapter|.
 
+                1. Otherwise, |promise| [=resolves=] with `null`.
```

Comment:
strictly, these two promise resolution lines should be on the content timeline, though we can clean this up later. I'm sure there are other examples of this elsewhere, I didn't look